apps/dh/api-dh/source/DataHub.WebApi.Tests/Integration/Controllers/WholesaleSettlementReportControllerTests.cs

```diff
@@ -37,17 +39,30 @@ public sealed class WholesaleSettlementReportControllerTests : ControllerTestsBa
         }
 
         [Fact]
-        public async Task CreateAsync_ReturnsOk()
+        public async Task DownloadAsync_ReturnsOk()
```

Comment:
praise: you could have taken the easy way out here, but you actually added a missing test instead. nice work!

Reply:
:)